components/staging/staging.html

```diff
@@ -15,7 +15,9 @@
         <textarea class="form-control" rows="2" placeholder="Body" data-bind="value: commitMessageBody, valueUpdate: 'afterkeydown', enable: !inRebase(), event: {keypress: onAltEnter}"></textarea>
         <div>
           <span class="amend" data-bind="visible: canAmend">
-            <span class="checkmark" data-bind="css: { checked: amend }">&#10003;</span>
+            <div class="checkmark" data-bind="click: toggleAmend, css: { checked: amend }">
```

Comment:
the click handler can be moved up to the `span.amend` element and removed from both `div.checkmark` and `a.amend-link`

Reply:
Yeah I think that would be a better idea